WalletWasabi.Fluent/ViewModels/Wallets/Settings/WalletCoinJoinSettingsViewModel.cs

```diff
@@ -78,13 +90,44 @@ private WalletCoinJoinSettingsViewModel(IWalletModel walletModel)
 					}
 				});
 
+		this.WhenAnyValue(x => x.SelectedOutputWalletName)
+			.Skip(1)
+			.ObserveOn(RxApp.TaskpoolScheduler)
+			.Subscribe(
+				x =>
+				{
+					_wallet.Settings.OutputWalletId = x.Wallet.WalletId;
+					_wallet.Settings.Save();
```

Comment:
`Save()` is not needed, it is not stored in the JSON file.

Reply:
Done